packages/compass-e2e-tests/tests/global-preferences.test.ts

```diff
@@ -194,7 +193,7 @@ describe('Global preferences', function () {
         expect(value).to.equal('false');
         expect(disabled).to.equal(''); // null = missing attribute, '' = set
         expect(bannerText).to.include(
-          'This setting cannot be modified as it has been set at Compass startup.'
+          'This setting cannot be modified as it has been set in the global Compass configuration file.'
```

Comment:
These changes are fine ofc, but was there anything that made them necessary? They don’t seem to be related to the rest of the PR, so I’m curious :)

Reply:
Like nothing important for the test result :) just noticed that similar tests in this file do it in another way and aligned them.